draft/2025-03-26-this-week-in-rust.md

```diff
@@ -244,22 +232,20 @@ Rusty Events between 2025-03-26 - 2025-04-23 🦀
     * [**Topic TBD**](https://www.meetup.com/rust-modern-systems-programming-in-leipzig/events/305741632)
 * 2025-04-15 | London, UK | [Women in Rust](https://www.meetup.com/women-in-rust/events/)
```

Comment:
Can we add this event in Stockholm on 2 Apr? https://www.meetup.com/stockholm-rust/events/306627608/?eventOrigin=group_upcoming_events

Reply:
Nevermind - this was already in the list, I failed to expand the compressed section in the PR diff.